sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -159,8 +159,8 @@ class DataFrameReader private[sql](sqlContext: SQLContext) extends Logging {
    * @param columnName the name of a column of integral type that will be used for partitioning.
    * @param lowerBound the minimum value of `columnName` used to decide partition stride
    * @param upperBound the maximum value of `columnName` used to decide partition stride
-   * @param numPartitions the number of partitions.  the range `minValue`-`maxValue` will be split
-   *                      evenly into this many partitions
+   * @param numPartitions the number of partitions, the range `lowerBound`-`upperBound` will be
```

Comment:
Agree. Do you need to specify whether the upper bound in inclusive/exclusive while you're at it? I also don't think this needs a JIRA BTW. There's no difference between the 'what' and 'how' here.